### PR TITLE
Update docs on how to create an experiment

### PR DESCRIPTION
**Pull request by safern**
cc: @jkotas @AaronRobinsonMSFT

### CODE REVIEW

**Review by jkotas — APPROVED** [review]
Thanks

File: CreateAnExperiment.md [hunk 1]
```diff
@@ -14,4 +14,6 @@ Experiments should be contained within a branch in the dotnet/runtimelab reposit
...
    - Edit `eng/pipelines/runtimelab.yml` in your branch to just build what your experiment needs on CI.
    - To avoid spurious github notifications for merges from upstream, delete `.github/CODEOWNERS` from your branch or replace it with setting specific to your experiment. Example: [#26](https://github.com/dotnet/runtimelab/pull/26/files)
+   - Make sure to edit your experiment branch to just build the packages that need to be built from that branch. [Example PR](https://github.com/dotnet/runtimelab/pull/467). To read more about why this can cause issues, read [this issue](https://github.com/dotnet/runtimelab/issues/465).
 - If your experiment is branched from [dotnet/runtimelab:standalone-template](https://github.com/dotnet/runtimelab/tree/standalone-template) follow the [README.md](https://github.com/dotnet/runtimelab/tree/standalone-template#standalone-experiments).
+- To make sure we follow our naming conventions, make sure all packages produced on your experiment are prefixed with Microsoft.* or System.*
```
Comment (AaronRobinsonMSFT) [inline CreateAnExperiment.md:19]: I would quite both "Microsoft.*" and "System.*" and then add the missing period.

**Review by AaronRobinsonMSFT — APPROVED** [review]
LGTM. Thanks!

File: CreateAnExperiment.md (hunk 1 above)
Comment (safern) [reply CreateAnExperiment.md:19]: Sounds good. I can add a back tick so that is shows nice in the markdown.

**Pull request merged**